assets/src/scss/layout/_country-selector.scss

```diff
@@ -0,0 +1,162 @@
+
+#country-selector {
+  --country-selector-- {
+    color: var(--site-footer--color, $white);
+    font-size: $font-size-sm;
+  }
+  margin-bottom: 24px;
+  text-align: start;
+  width: 100%;
+
+  a --country-selector--a-- {
+    color: var(--site-footer--a--color, $white);
+    text-decoration: underline;
+
+    @include medium-and-up {
+      text-decoration: none;
+    }
+  }
+
+  .control-toggle {
+    --country-selector--toggle-- {
+      background: transparent;
+      border: none;
+      border-bottom: 1px solid rgba(255, 255, 255, 0.5);
+      color: var(--site-footer--color);
+      text-align: start;
+    }
+    cursor: pointer;
+    height: 72px;
+    line-height: 72px;
+    padding-bottom: 24px;
+    white-space: nowrap;
+    width: 100%;
+
+    &::before {
+      --country-selector--logo-- {
+        background-color: var(--site-footer--color, $white);
+      }
+      background-repeat: no-repeat;
+      content: "";
+      display: inline-block;
+      height: 22px;
+      margin-inline-end: 8px;
+      mask-image: url("../../images/gp-g-logo.svg");
+      transition: transform 300ms linear;
+      vertical-align: middle;
+      width: 16px;
+    }
+
+    &::after {
+      --country-selector--chevron-- {
+        background-color: var(--site-footer--color, $white);
+      }
+      background-repeat: no-repeat;
+      content: "";
+      display: inline-block;
+      height: 8px;
+      margin-inline-start: 8px;
+      mask: url("../../images/chevron-down.svg") 0 0/8px 8px;
```

Comment:
Is this different than the existing [chevron icon](https://github.com/greenpeace/planet4-master-theme/blob/master/assets/src/images/icons/chevron.svg)? Maybe we can use the same and just rotate it?

Reply:
Yep :+1: working on fixes right now, it's in it :)